Added session "Design for intelligence: Apps, evolved" (#67)
Blackjacx/WWDC: summaries/2020/10086.md

https://developer.apple.com/wwdc20/10086

Presenter: _Mark Mikin_

- This is one of those "why" sessions. Why we should build something?
- **Intelligent System Experience**
  - The main idea is to be proactive
  - Intelligence is how the OS works with the apps that people use every day to make the "every day" easier for people
  - Intelligence is design (it should be viewed as a design practice)
- **Living design**. What is that?
  - The core job of the designer is to help people accomplish something

>   - One of the key was a designer enables this is is be leveraging the familiar

Reviewer: This sentence is very confusing, I'm not sure what you mean 😅  Maybe this?

```suggestion
  - One of the key ways a designer can enable this is by leveraging the familiar
```

The "familiar" what, though? 🤔

Author: Oh yes. I just completely messed and missed it. This sentence is related to the first one. So idk if it can be combined and/or rephrased to show that a designer role is to help people to accomplish something by leveraging the familiar stuff that people used to.
That moment: [2:14](https://developer.apple.com/videos/play/wwdc2020/10086/)

Reviewer: Got it!

>   - One of the key ways a designer can enable this is by leveraging the familiar

Reviewer: ```suggestion
  - One of the key ways a designer can leverage this is by using familiar elements
```

Reviewer: Or:

```suggestion
  - One of the key ways a designer can leverage this is by using elements familiar to the user
```

Author: Second one sounds great to me. Thank you!
